R/adapter.r

```diff
@@ -151,9 +151,11 @@ construct_file_adapter <- function() {
     if ('.rds' == substring(opts$resource, nchar(opts$resource) - 3, nchar(opts$resource)))
       readRDS(opts$resource)
     else {
+      test_if_url <- function(str) grepl("^https?://", str)
+      filesource <- ifelse(test_if_url(opts$resource), getURL(opts$resource), opts$resource)
```

Comment:
Ahh, I see. It gets passed to `file` below when it should be passed to `text`. I thought I had tested the file adapter and it worked with URLs. :/

Doesn't matter! I will re-factor this and make syberiaStages obsolete today.